e2e/testsuite/relayer.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	cosmosRelayerRepository = "ghcr.io/cosmos/relayer"
+	cosmosRelayerRepository = "damiannolan/rly"
```

Comment:
This is a custom build pushed to dockerhub for temporary use. It includes type registration of the `09-localhost` ClientState so that the relayer can continue to function as normal when querying all client states from a node.

Reply:
Type registration in the relayer is performed [here](https://github.com/cosmos/relayer/blob/main/relayer/chains/cosmos/module/app_module.go#L25-L27)